README.md

```diff
@@ -175,6 +176,33 @@ npm install
 npm run test
 ```
 
+<h2 id="docker-local-development">
+  Local Development using Docker (Dev Containers)
+</h2>
+
+1. Install docker https://www.docker.com/.
+2. Install [Microsoft VS Code](https://code.visualstudio.com/Download)
+3. Clone this repo.
+4. Open this repo in VS Code.
+5. Install the Dev Container extension.
```

Comment:
We can add the extension ID so consumers can install the exact extension you're talking about. 
```suggestion
5. Install the Dev Container extension. In the `Extensions` tab, you can search: `ms-vscode-remote.remote-containers`.
```